app/test_main.py

```diff
@@ -1 +1,14 @@
 # write your code here
+from app.main import can_access_google_page
+from unittest import mock
+
+
+def test_can_access_google_page():
+    with (
+        mock.patch("current_time") as mocked_current_time,
+        mock.patch("response") as mocked_response
+    ):
+        mocked_current_time = True
+        mocked_response.status_code = 200
+
+        assert can_access_google_page == "Accessible"
```

Comment:
You are comparing a function reference 'can_access_google_page' with a string 'Accessible'. You likely intended to call the function instead. Remember to call the function with necessary arguments.